src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Declarations_AllDeclarations.cs

```diff
@@ -27,6 +34,27 @@ public static partial class SymbolFinder
             return declarations.SelectAsArray(t => t.Symbol);
         }
 
+        /// <summary>
+        /// Find the declared symbols from either source, referenced projects or metadata assemblies with the specified name.
+        /// </summary>
+        public static async Task<IEnumerable<ISymbol>> FindDeclarationsAsync(
+            Project project, string name, bool ignoreCase, SymbolFilter filter, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            var declarations = await FindAllDeclarationsAsync(project, name, ignoreCase, filter, cancellationToken).ConfigureAwait(false);
+            return declarations.SelectAsArray(t => t.Symbol);
+        }
+
+        #endregion
+
+        #region Current API
+
+        // This region contains the current FindDeclaratins APIs.  The current APIs allow for OOP 
+        // implementation and will defer to the oop server if it is available.  If not, it will
+        // compute the results in process.
+
+        /// <summary>
+        /// Find the declared symbols from either source, referenced projects or metadata assemblies with the specified name.
+        /// </summary>
         internal static async Task<ImmutableArray<SymbolAndProjectId>> FindAllDeclarationsAsync(
```

Comment:
Ideally, sure.  But that would be a ton of work (especially given that SymbolAndProjectId is not public).  It is out of scope of this work (afaict) to make this public, and i would not want to gate the OOP work on that.